recipe/meta.yaml

```
@@ -1,44 +1,61 @@
-{% set version = "0.4.23" %}
+{% set version = "0.4.35" %}
 {% set name = "jaxlib" %}
 
 package:
   name: jaxlib
   version: {{ version }}
 
 source:
-  url: https://github.com/google/jax/archive/{{ name }}-v{{ version }}.tar.gz
-  sha256: e4c06d62ba54becffd91abc862627b8b11b79c5a77366af8843b819665b6d568
+  url: https://github.com/google/jax/archive/jax-v{{ version }}.tar.gz
+  sha256: 65e086708ae56670676b7b2340ad82b901d8c9993d1241a839c8990bdb8d6212
+  patches:
+      - patches/0000-Consolidated-build-fixes-for-XLA.patch  # [not win]
 
 build:
-  number: 1
+  number: 0
   # s390x is missing bazel.
-  skip: true  # [s390x or py<39]
+  skip: true  # [s390x or py<310]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - clang    # [win]
+    - llvmdev  # [win]
+    - lld      # [win]
     - python
-    # https://github.com/google/jax/blob/jaxlib-v0.4.23/build/build.py#L207
-    # https://github.com/google/jax/blob/jaxlib-v0.4.23/build/build.py#L98
-    - bazel >=5.1.1,<7
+    # https://github.com/jax-ml/jax/blob/jax-v0.4.35/build/build.py#L198
+    - bazel >=6.5.0,<7
     - bazel-toolchain >=0.1.9  # [not win]
+    - libabseil {{ libabseil }}
+    - patch  # [not win]
+    - libgrpc {{ libgrpc }}
+
   host:
     - python
     - pip
     - wheel
     - setuptools
-    - numpy 1.22.*  # [py<311]
-    - numpy 1.23.*  # [py==311]
-    - numpy 1.26.*  # [py>=312]
+    - numpy 2
     - python-build
+    - libabseil {{ libabseil }}
+    - openssl {{ openssl }}
+    - nsync 1.24.0
+    - snappy 1.2.1
+    - double-conversion 3.1.5
+    - jsoncpp 1.9.4
+    - pybind11 2.13.6
+    - libgrpc {{ libgrpc }}
+    - libcurl {{ libcurl }}  # [not win]
+    - zlib {{ zlib }}
```

```suggestion
    - libabseil {{ libabseil }}  # [not win]
    - openssl {{ openssl }}  # [not win]
    - nsync 1.24.0
    - snappy 1.2.1  # [not win]
    - double-conversion 3.1.5 # [not win]
    - jsoncpp 1.9.4  # [not win]
    - pybind11 2.13.6
    - libgrpc {{ libgrpc }}  # [not win]
    - libcurl {{ libcurl }}  # [not win]
    - zlib {{ zlib }}  # [not win]
```
Most host dependencies are not needed on windows, since TF_SYSTEM_LIBS does not work.